code/modules/vehicles/segway.dm

```diff
@@ -6,6 +6,10 @@
 	callme = "secway"
 	anchored = 0 //This is not so heavy
 
+	Move()
+		playsound(src.loc, 'sound/effects/secway.ogg', 75, 1)
```

Comment:
No, but I cut it down enough that it wasn't awfully fucky and it didn't seem to cut out other sounds.